homeassistant/components/indevolt/strings.json

```
@@ -0,0 +1,129 @@
+        },
+        "data_description": {
+          "host": "[%key:component::indevolt::config::step::user::data_description::host%]",
+          "port": "The TCP port of the Indevolt device (default: 8080).",
+          "scan_interval": "[%key:component::indevolt::config::step::user::data_description::scan_interval%]",
+          "model": "The model of the Indevolt device."
+        }
+      }
+    },
+      "error": {
+      "timeout": "[%key:common::config_flow::error::timeout_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+      "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "reconfigure_successful": "[%key:common::config_flow::abort::reconfigure_successful%]",
+      "another_device": "The configured device is not the same found on this IP address"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "dc_input_power1": {
+        "name": "DC Input Power1"
+      },
+      "dc_input_power2": {
+        "name": "DC Input Power2"
+      },
+      "total_ac_output_power": {
+        "name": "Total AC Output Power"
+      },
+      "daily_production": {
+        "name": "Daily Production"
+      },
+      "cumulative_production": {
+        "name": "Cumulative Production"
+      },
+      "total_ac_input_power": {
+        "name": "Total AC Input Power"
+      },
+      "total_ac_input_energy": {
+        "name": "Total AC Input Energy"
+      },
+      "total_dc_output_power": {
+        "name": "Total DC Output Power"
+      },
+      "battery_power": {
+        "name": "Battery Power"
+      },
+      "battery_soc": {
+        "name": "Battery SOC"
+      },
+      "emergency_power_supply": {
+        "name": "Emergency power supply"
+      },
+      "battery_daily_charging_energy": {
+        "name": "Battery Daily Charging Energy"
+      },
+      "battery_daily_discharging_energy": {
+        "name": "Battery Daily Discharging Energy"
+      },
+      "battery_total_charging_energy": {
+        "name": "Battery Total Charging Energy"
+      },
+      "battery_total_discharging_energy": {
+        "name": "Battery Total Discharging Energy"
+      },
+      "meter_power": {
+        "name": "Meter Power"
+      },
+      "working_mode": {
+        "name": "Working mode"
+      },
+      "battery_charge_discharge_state": {
+        "name": "Battery Charge/Discharge State"
+      },
+      "meter_connection_status": {
+        "name": "Meter Connection Status"
+      },
+      "dc_input_power3": {
+        "name": "DC Input Power3"
+      },
+      "dc_input_power4": {
+        "name": "DC Input Power4"
+      },
+      "rated_capacity": {
+        "name": "Rated capacity"
+      },
+      "bypass_power": {
+        "name": "Bypass Power"
+      }
+    }
+  },
+  "exceptions": {
+    "update_failed": {
+      "message": "Unable to fetch data."
+    }
+  }
+}
```

Missing blank line at end of file.


```suggestion
}

```